examples/uptime-monitor/steps/dashboard.step.js

```diff
@@ -0,0 +1,158 @@
+        }
+        .stat-value {
+            font-size: 2rem;
+            font-weight: bold;
+            margin-bottom: 5px;
+        }
+        .website-card {
+            background: #333;
+            padding: 20px;
+            margin: 10px 0;
+            border-radius: 10px;
+            border-left: 4px solid #28a745;
+        }
+        .website-card.down {
+            border-left-color: #dc3545;
+        }
+        .website-url {
+            font-size: 1.2rem;
+            margin-bottom: 10px;
+        }
+        .website-details {
+            display: flex;
+            justify-content: space-between;
+            font-size: 0.9rem;
+            color: #ccc;
+        }
+    </style>
+</head>
+<body>
+    <div class="container">
+        <div class="header">
+            <h1>📊 Uptime Monitor</h1>
+            <p>Real-time website monitoring dashboard</p>
+        </div>
+
+        <div class="stats">
+            <div class="stat-card">
+                <div class="stat-value" style="color: #17a2b8;">${summary.total}</div>
+                <div>Total Sites</div>
+            </div>
+            <div class="stat-card">
+                <div class="stat-value" style="color: #28a745;">${summary.up}</div>
+                <div>Online</div>
+            </div>
+            <div class="stat-card">
+                <div class="stat-value" style="color: #dc3545;">${summary.down}</div>
+                <div>Offline</div>
+            </div>
+            <div class="stat-card">
+                <div class="stat-value" style="color: #6f42c1;">${summary.avgUptime}%</div>
+                <div>Avg Uptime</div>
+            </div>
+        </div>
+
+        <div class="websites">
+            ${websites.length === 0 ? 
+                '<div style="text-align: center; padding: 40px;"><h3>No websites monitored yet</h3></div>' :
+                websites.map(site => `
+                    <div class="website-card ${site.status === 'down' ? 'down' : ''}">
+                        <div class="website-url">
+                            ${site.status === 'up' ? '✅' : '❌'} ${site.website}
+                        </div>
+                        <div class="website-details">
+                            <span>Uptime: ${site.uptime}%</span>
+                            ${site.responseTime ? `<span>Response: ${site.responseTime}ms</span>` : ''}
+                            <span>Last checked: ${new Date(site.timestamp).toLocaleString()}</span>
+                        </div>
+                    </div>
+                `).join('')
+            }
+        </div>
+
+        <div style="text-align: center; margin-top: 30px; color: #666;">
+            <p>Powered by Motia Backend Framework</p>
+            <p>Last updated: ${new Date().toLocaleString()}</p>
+        </div>
+    </div>
+
+    <script>
+        // Auto-refresh every 30 seconds
+        setTimeout(() => location.reload(), 30000);
+    </script>
+</body>
+</html>`;
```

Comment:
major-concern: this is not something we would want to show as an example in our catalog

use this repository as example:
https://github.com/MotiaDev/chessarena-ai

- we don't want to serve html in motia.
- we don't want polling strategy, instead, you could use [streams](https://www.motia.dev/docs/concepts/streams)